### DISCUSSION TITLE
How do I reset the code block css

### DISCUSSION THREAD

**Discussion by nphalem** [BryanJenksCommunity/FAQ#172] [Q&A] [answered]
Hey, I'm trying to figure out how to remove these orange lines in code blocks as shown in picture

![Screenshot_20230519_100220](https://github.com/BryanJenksCommunity/FAQ/assets/98975507/6cf20ebe-422f-4572-886a-8d86aa8e3c9e)

I still want to keep the line numbers though.
Also is the theme for syntax highlighting a gruvbox theme?

**Comment by tallguyjenks** [comment, marked as answer]
the syntax highlight is also affected a plugin i believe, i think its the combo of `codemirror options` and `style settings` and the code fence CSS is the section you'll get if you search for `2.15.4`
